File: packages/expo-development-client/android/src/main/java/expo/modules/developmentclient/DevelopmentClientPackage.java
package expo.modules.developmentclient;

import java.util.Arrays;
import java.util.Collections;
import java.util.List;

import com.facebook.react.ReactPackage;
import com.facebook.react.bridge.NativeModule;
import com.facebook.react.bridge.ReactApplicationContext;
import com.facebook.react.uimanager.ViewManager;

public class DevelopmentClientPackage implements ReactPackage {
  @Override
  public List<NativeModule> createNativeModules(ReactApplicationContext reactContext) {
    return Arrays.asList(
        new DevelopmentClientModule(reactContext),
        new DevelopmentClientDevMenuExtensions(reactContext)
    );
  }

  @Override
  public List<ViewManager> createViewManagers(ReactApplicationContext reactContext) {
    return Collections.emptyList();
  }
}

File: packages/expo-development-client/android/src/main/java/expo/modules/developmentclient/DevelopmentClientPackage.kt
package expo.modules.developmentclient

import com.facebook.react.ReactPackage
import com.facebook.react.bridge.NativeModule
import com.facebook.react.bridge.ReactApplicationContext
import com.facebook.react.uimanager.ViewManager
import expo.modules.developmentclient.modules.DevelopmentClientDevMenuExtensions
import expo.modules.developmentclient.modules.DevelopmentClientModule

class DevelopmentClientPackage : ReactPackage {
  override fun createNativeModules(reactContext: ReactApplicationContext): List<NativeModule> =
    listOf(
      DevelopmentClientModule(reactContext),
      DevelopmentClientDevMenuExtensions(reactContext)
    )

  override fun createViewManagers(reactContext: ReactApplicationContext): List<ViewManager<*, *>> = emptyList()
}
